### PR TITLE
Remove `Rack::Multipart` dependency on `Rack::Request`.

### PR DESCRIPTION
**Pull request by ioquatix**

### CODE REVIEW

File: test/spec_request.rb [hunk 1]
```diff
@@ -1867,8 +1872,6 @@ def initialize(req)
       end
 
       def delegate?; true; end
-
-      def env; @req.env.dup; end
```
Comment (ioquatix) [inline test/spec_request.rb:1871]: @tenderlove since you wrote this code do you mind explaining why you are calling dup here if you recall the reason?

**Comment by ioquatix** [comment]
@jeremyevans my understanding is even if you approve a PR, if there is open unresolved conversations, they will block merging. So my understanding is, you can add feedback, approve the PR, and it can be merged provided those are resolved.

File: test/spec_request.rb (hunk 1 above)
Comment (tenderlove) [reply test/spec_request.rb:1871]: IIRC the idea was that if we return the hash directly, then people can muck with it without going through getters / setters.  Looks like this is in a spec though so, I'm not sure why we need it. 😅

**Review by tenderlove — APPROVED** [review]

File: test/spec_request.rb (hunk 1 above)
Comment (ioquatix) [reply test/spec_request.rb:1871]: Yeah, IMHO it was hiding bugs.

**Pull request merged**